Codec/Archive/Zip/Internal.hs

```diff
@@ -271,12 +294,16 @@ predictComment original xs =
 -- that re-create those entires.
 
 toRecreatingActions
-  :: Path Abs File     -- ^ Name of archive file where entires are found
+  :: Either Handle (Path Abs File)     -- ^ Name of archive file where entires are found
   -> Map EntrySelector EntryDescription -- ^ Actual list of entires
   -> Seq PendingAction -- ^ Actions that recreate the archive entries
-toRecreatingActions path entries = E.foldl' f S.empty (M.keysSet entries)
+toRecreatingActions (Right path) entries = E.foldl' f S.empty (M.keysSet entries)
   where f s e = s |> CopyEntry path e e
+toRecreatingActions (Left _) entries = E.foldl' f S.empty (M.keysSet entries)
+  where f s e = s |> CopyEntry blindPath e e
+
 
+                
```

Comment:
I will have a look at all of the items and correct. I have no
preferences for space, or naming, so what you suggest I can make it so.
On Sat, 2016-08-13 at 06:31 -0700, Mark Karpov wrote:
> In Codec/Archive/Zip/Internal.hs:
> >  
> > +                
> Extra lines again. Can you stage selectively?
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.